concent_api/core/payments/bankster.py

```diff
@@ -51,11 +58,14 @@ def __init__(
 
 
 def claim_deposit(
+    subtask_id: str,
     concent_use_case: ConcentUseCase,
     requestor_ethereum_address: str,
     provider_ethereum_address: str,
     subtask_cost: int,
-) -> Tuple[bool, bool]:
+    requestor_public_key: bytes,
+    provider_public_key: bytes,
+) -> Tuple[Optional[DepositClaim], Optional[DepositClaim]]:
```

Comment:
This needs to be updated in the blueprint. When you change stuff like this, please always remember to drop me a comment in the blueprint issue (and it would be a good idea to make a comment in the issue this pull request is solving too).

Reply:
OK. So just please remember about it when we close this pull request. I'm leaving this as unresolved as a reminder.